src/transformers/modeling_flax_utils.py

```diff
@@ -97,6 +97,7 @@ def __init__(
         input_shape: Tuple = (1, 1),
         seed: int = 0,
         dtype: jnp.dtype = jnp.float32,
+        do_init: bool = True,
```

Comment:
```suggestion
        _do_init: bool = True,
```

Let's maybe make it experimental for now no?

Reply:
We also have 0 docs here (even before this PR) should we maybe add some docs at some point?

Reply:
We should definitely add docs for this. Will probably also need a new doc page to explain this new feature.
I will work on it in follow-up PRs.